.envrc

```diff
@@ -0,0 +1,7 @@
+# Automatically sets up your devbox environment whenever you cd into this
+# directory via our direnv integration:
+
+eval "$(devbox generate direnv --print-envrc)"
```

Comment:
I would move this into the `shellHook` of the Nix shell, and replace the content of `.envrc` with:

```bash
use flake

watch_file nix/devshell.nix
```

This is currently what I have in my local `.envrc` and not doing `use flake` here would likely make the Nix shell with `direnv` unusable. This also has the advantage of setting up `devbox` for you when using `direnv`.